packages/block-library/src/subhead/index.js

```diff
@@ -18,7 +18,7 @@ export const settings = {
 
 	description: __( 'This block is deprecated. Please use the Paragraph block instead.' ),
 
-	icon: <svg role="img" aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M7.1 6l-.5 3h4.5L9.4 19h3l1.8-10h4.5l.5-3H7.1z" /></svg>,
+	icon: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M7.1 6l-.5 3h4.5L9.4 19h3l1.8-10h4.5l.5-3H7.1z" /></svg>,
```

Comment:
I get that we don't *have* to use our component directly, but I wonder why we wouldn't. This is less explicit and we're already changing all of the lines 🤷‍♂️

Reply:
I don't mind, I was hesitant as well.

Reply:
As mentioned though, feel free to leave them SVG if you add docs. I think that's probably the better approach, upon reflection.